src/types.ts

```diff
@@ -26,6 +26,7 @@ export interface AdvancedParametersBase {
   displaySearchSuggestionResultCounts?: boolean;
   debounce?: number;
   translations?: Translations;
+  fetchZeroStateOnFocus?: boolean;
```

Comment:
would having two props instead of a single one that replaces load with focus events be better?
ex:
fetchZeroStateOnFocus: default false
fetchZeroStateOnInit: default true

Reply:
Lets discuss this with the team

Reply:
After team discussion, we will leave the implementation as is with a single option, having two options may lead to scenarios where both are TRUE and that will cause duplicated tracking due to the wrong configuration